src/components/molecules/county-dropdown.tsx

```diff
@@ -14,8 +14,8 @@ const countyOptions: CountyOption[] = [];
 
 for (const l of counties) {
   countyOptions.push({
-    label: l.county,
-    value: l.county
+    label: l.label ? l.label : l.county,
```

Comment:
```suggestion
    label: l.label || l.county,
```

Reply:
Made these changes, need @floridemai to review again for some further changes

Reply:
Thanks @JohnAGill - if you're waiting for  don't forget to press the re-review button next to Paul's name: 
![image](https://user-images.githubusercontent.com/29628323/91992532-2860d500-ed2c-11ea-83ae-bbbf0e856494.png)